hack/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM ubuntu:artful
```

Comment:
maybe consider going straight to bionic?

Reply:
Docker is not yet support on bionic. :(.